test/meuse_comp.jl

```diff
@@ -0,0 +1,54 @@
+@testset "Meuse R gstat compatibility" begin
+    """meuse_grid.csv constains (x, y) plus estimation/variance of log(zinc)
+    for global and local (25 samples)
+    """
+    datadir = joinpath(@__DIR__,"data")
+    data_meuse = readgeotable(joinpath(datadir,"meuse.csv"), coordnames=(:x,:y))
+    grid_meuse = readgeotable(joinpath(datadir,"meuse_grid.csv"), coordnames=(:x,:y))
+
+    dim = 2
+    tol = 0.001
+    knn = 25
+
+    vmodel = SphericalVariogram(range=874.0,nugget=0.04,sill=0.59+0.04)
+    problem = EstimationProblem(data_meuse, grid_meuse, :log_zinc)
```

Comment:
Given that grid_meuse is a spatial data, I am surprised that the EstimationProblem didn't throw an error. The second argument is supposed to be a domain (no data).

Reply:
Well, any data with coordinates should be a domain. Any case, if this is an issue you can report it :-) and it would be fixed in a different PR.

Reply:
I think to make the code clear for future readers and maintainers, we should load the data from file as `Restimates` or something, and then create the problem with `domain(Restimates)`.

But you are right that our conceptual understanding of the data is domain + table of values, so a data in a sense satisfy the domain traits. Thank you for spending the time to figure out the codebase, it is under constant refactoring to be more general, and I noticed you picked the recent change `npoints` to `nelms`. :100:

Reply:
Do you mean a variable  (Restimates) starting with an upper case? Is it a good convention?

Reply:
Sorry the example name I chosen was terrible. The idea is to use some good name that represents "data" and then at the estimation problem we call `domain(whatevername)`. This way we signal the maintainer that what we are passing is indeed a domain and not the data attached to it.